.github/workflows/release.yml

```diff
@@ -24,24 +27,27 @@ jobs:
       - uses: actions/checkout@v2
 
       # Decrypt the ssh secret
-      - name: Decrypt files
+      - name: Decrypt files and set constants
         env:
           SSH_KEY: ${{ secrets.SSH_KEY }}
           MOZILLA_API_SECRET: ${{ secrets.MOZILLA_API_SECRET }}
           MOZILLA_API_KEY: ${{ secrets.MOZILLA_API_KEY }}
-          GHWS: ${{ github.workspace }}
-          # We use version 0.0.0 for testing this script, becuase github.event.release.tag_name
-          # only exists if the release event has fired
-          TAG_NAME: ${{ github.event.release.tag_name } || {'0.0.0'}}
+          TAG_NAME: ${{ github.event.release.tag_name }}
+        shell: bash
         run: |
-          echo "$SSH_KEY" | base64 -d > ./$GHWS/scripts/ssh_key
-          chmod 600 ./$GHWS/scripts/ssh_key
+          echo "$SSH_KEY" > ./scripts/ssh_key
+          chmod 600 ./scripts/ssh_key
+          ls -al ./scripts/
```

Comment:
This line was meant for debug, I suppose, and could be removed now

Reply:
OK, done.